src/MonitoringConfiguration.ts

```diff
@@ -67,6 +71,8 @@ export type Config = {
   metagraph: MetagraphProps;
   network: NetworkProps;
   check_healthy_interval_in_minutes: number;
+  minimal_disk_space_in_gib?: number;
```

Comment:
`minimal` isn't quite correct english. How about just `min`? Or else `minimum`. 

Is it possible to make this a percentage instead though? I think that would be more useful since it wouldn't depend on the current disk size. 

```suggestion
  min_disk_space_percent?: number;
```

Reply:
Sure, I'll change thanks 👍🏼